R/occDetModel.r

```diff
@@ -168,11 +176,15 @@
 #'                        region_aggs = list(agg1 = c('region1', 'region2')))       
 #' }
 #' @export
-#' @references Roy, H.E., Adriaens, T., Isaac, N.J.B. et al. (2012) Invasive alien predator
-#'             causes rapid declines of native European ladybirds. Diversity & Distributions,
-#'             18, 717-725.
+#' @references Isaac, N.J.B., van Strien, A.J., August, T.A., de Zeeuw, M.P. and Roy, D.B. (2014).
+#'             Statistics for citizen science: extracting signals of change from noisy ecological data.
+#'             \emph{Methods in Ecology and Evolution}, 5: 1052-1060.
+#' @references Outhwaite, C.L., Chandler, R.E., Powney, G.D., Collen, B., Gregory, R.D. & Isaac, N.J.B. (2018).
+#'             Prior specification in Bayesian occupancy modelling improves analysis of species occurrence data. 
+#'             \emph{Ecological Indicators}, 93: 333-343.
 
-occDetModel <- function(taxa, site, survey,
+occDetModel <- function(taxa, site, survey, 
+                        replicate = NULL, closure_period = NULL, criterion = 1, provenance = NULL,
```

Comment:
@drnickisaac Its a little confusing to have the parameters in one order in the help doc (@params above), and in a different order in the call (here), I suggest re-ordering here.